en/mavgen_c/README.md

```diff
@@ -133,3 +283,14 @@ if (mavlink_parse_char(MAVLINK_COMM_0, buf[i], &msg, &status)) {
 	}
 }
 ```
+
+## Examples
+
+The following examples show the use of the API.
+
+* [UART Interface](../examples/c_uart.md): Simple C example of a MAVLink to UART interface for Unix-like systems.
+* [UDP Example](../examples/c_udp.md): Simple C example of a MAVLink UDP interface for Unix-like systems (Linux, MacOS, BSD, etc.).
+
+In addition, the C library is used in numerous open source systems:
+- [PX4](https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_receiver.cpp)
+- [ArduPilot](https://github.com/ArduPilot/ardupilot/blob/master/libraries/GCS_MAVLink/GCS_Common.cpp)
```

Comment:
You could list the Dronecode SDK here:
https://github.com/Dronecode/DronecodeSDK/blob/develop/core/mavlink_receiver.cpp#L28-L51